apps/src/music/Instructions.jsx

```diff
@@ -0,0 +1,168 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default class Instructions extends React.Component {
```

Comment:
I'd be interested to hear if you all are hoping to replace instructions with a pattern like this or if this is intended to be only for music lab

Reply:
This is just a temporary component that can be replaced by a real instructions component once we integrate with our level system.